extensions/sql-migration/src/api/utils.ts

```diff
@@ -279,16 +279,15 @@ export function getMigrationStatusWithErrors(migration: azure.DatabaseMigration)
 	warningCount += properties.migrationFailureError?.message?.length > 0 ? 1 : 0;
 
 	// file upload blocking errors
-	warningCount += properties.migrationStatusDetails?.fileUploadBlockingErrors?.length ?? 0;
+	warningCount += properties.migrationStatusWarnings?.fileUploadBlockingErrorCount ?? 0;
 
 	// restore blocking reason
-	warningCount += properties.migrationStatusDetails?.restoreBlockingReason ? 1 : 0;
+	warningCount += properties.migrationStatusWarnings?.restoreBlockingReason ? 1 : 0;
 
-	// sql data copy errors
-	warningCount += properties.migrationStatusDetails?.sqlDataCopyErrors?.length ?? 0;
+	// complete restore error message
+	warningCount += properties.migrationStatusWarnings?.completeRestoreErrorMessage ? 1 : 0;
```

Comment:
should this also check for string.length > 0 to assert it is a warning

Reply:
this is what I meant to do, thanks for catching that